cms/djangoapps/contentstore/views/videos.py

```diff
@@ -338,7 +384,8 @@ def videos_index_json(course):
             "client_video_id": "video.mp4",
             "created": "1970-01-01T00:00:00Z",
             "duration": 42.5,
-            "status": "upload"
+            "status": "upload",
+            "course_video_image_url": "https://video/images/1234.jpg"
```

Comment:
In case of S3 url will be an absolute cloudfront url. For devstack it will be a relative url.